CHANGELOG.asciidoc

```diff
@@ -33,6 +33,8 @@ https://github.com/elastic/beats/compare/v1.3.0...1.3[Check the HEAD diff]
 
 *Filebeat*
 
+- Fix open file handler issue {issue}2028[2028]
```

Comment:
close newly opened files faster on error. (file closing is backed by gc by default)